udata/utils.py

```diff
@@ -199,9 +200,10 @@ def recursive_get(obj, key):
     return recursive_get(value, parts) if parts else value
 
 
-def unique_string(length=None):
+def unique_string(length=36):
     '''Generate unique string'''
-    string = str(uuid4())
+    # We need a string longer than length and an UUID is 36 chars length
+    string = str(uuid4()) * int(math.ceil(length / 36.0))
```

Comment:
`* 2` is not sufficient, we need at least 3.
But I spent too much time searching the cause of the test failure so lets make it work with any length.